tools/i18n/extract.php

```diff
@@ -9,7 +9,7 @@
  */
 class StringExtractor {
 
-	var $rules = array(
+	var $rules          = array(
```

Comment:
Not sure if we have to update tools/i18n/ here too. 🤷‍♂️

Did you run the tests manually?

Reply:
That's the expected behaviour, but it looks a bit weird. Move the `$comment_prefix` declaration above `$rules`, or add a blank line between them.

Run what tests manually?

Reply:
I meant the whole `tools/i18n` directory.

The tests in https://core.trac.wordpress.org/browser/trunk/tools/i18n/t.

Reply:
Given that I've never opened that directory before... no. Why doesn't it run in Travis?

Reply:
I was never a fan of having that directory on core.trac at all. For me the canonical source is still https://i18n.trac.wordpress.org/browser/tools.

Reply:
Let's just external one of them.